### PR TITLE
feat(Jest) : init unit test using Jest & RTL

### PR DESCRIPTION
**Pull request by Ar-mane**

### CODE REVIEW

**Comment by Ar-mane** [comment]
image

File: package.json [hunk 1]
```diff
@@ -5,21 +5,37 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
```
Comment (Ar-mane) [inline package.json]: maybe no need for that , recheck

File: package.json [hunk 2]
```diff
@@ -5,21 +5,37 @@
...
+    "@types/jest": "^27.4.0",
+    "@types/node": "17.0.13",
     "@types/react": "17.0.38",
+    "babel-jest": "^27.4.5",
     "babel-plugin-styled-components": "^2.0.2",
     "eslint": "8.7.0",
-    "eslint-config-next": "12.0.8",
+    "eslint-config-next": "12.0.9",
+    "jest": "^27.4.5",
+    "jest-transform-stub": "^2.0.0",
```
Comment (Ar-mane) [inline package.json]: to remove

File: jest/jest.setup.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import '@testing-library/jest-dom';
+jest.mock('next/image', () => ({
+    __esModule: true,
+    default: () => {
+        return 'Next image stub';
```
Comment (Ar-mane) [inline jest/jest.setup.js]: a nice string maybe

File: components/mainscreen/__test__/StartMenu.test.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import StartMenus from "@components/startMenu/StartMenu";
+
+import { render } from "@testing-library/react";
```
Comment (Ar-mane) [inline components/mainscreen/__test__/StartMenu.test.tsx]: change root directory

**Review by Djinzo — APPROVED** [review]

**Pull request merged**